### DISCUSSION TITLE
Improve Playlist to be able to cycle through dashboards with custom settings

### DISCUSSION THREAD

**Discussion by PeterZaitsev** [grafana/grafana#49775] [Dashboards, Panels & Visualizations]
Currently Playlist is limited - you're only able to cycle through dashboard with their default settings which is not very useful for templated dashboard.   It would be very good if I could specify variable for iteration so I can iterate for example though all the hosts

**Comment by RandomUser0815** [comment]
I agree. I thought I clone my dashboards and just run through them with different hostnames. But I was mistaken.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Reply by RandomUser0815** [reply]
just look in #773 - there are more than enough people hoping for this feature.
